Implement the `findroute`, `findroutetonode` and `findroutebetweennodes` commands

mohitk108 opened the pull request:
The implementations of these commands are added through this PR.
Code for autocompletion and unit tests, for the respective commands are also added.
To see the documentation of these commands, kindly refer : [API Documentation](https://acinq.github.io/eclair/#route)

Commit:
Implement the `findroute` command

Commit:
Implement the `findroutetonode` command

t-bast commented on src/nativeTest/kotlin/mocks/EclairClientMocks.kt:
```diff
@@ -879,6 +891,18 @@ class DummyEclairClient(
   ]
 }
 """
+        val validFindRouteToNodeResponse = """{
```
You're only testing the case where `format=nodeId`, you need to also test the `format=shortChannelId` and `format=full` cases.

mohitk108 replied:
Yes, I'll cover them soon.

Commit:
Implement the `findroutebetweennodes` command

Commit:
Add test for format=nodeId and shortChannelId

mohitk108 commented:
I've added the test for different cases, like `format=nodeId` and `sshortChannelId` as well in 9f6f5fa7fa5b18fadf59d45629a9412b740ccf73 for the `findroute` command.
The build will fail right now, since I've modified the EclairClientMocks. I will update the remaining tests once you approve this particular change.
You can individually run the test right now. It's passing for me.

t-bast commented:
> I will update the remaining tests once you approve this particular change.

I'm not sure what particular change you want me to review before finishing the work?
You should simply finish updating the tests and then I'll be able to review it?
